packages/sanity/src/core/field/__workshop__/index.ts

```diff
@@ -2,8 +2,8 @@ import {defineScope} from '@sanity/ui-workshop'
 import {lazy} from 'react'
 
 export default defineScope({
-  name: 'sanity/field',
```

Comment:
great call changing from `sanity` => `core` here

Reply:
Thanks @bjoerge – that's done!